schema/complextype.cpp

```diff
@@ -232,6 +233,18 @@ bool ComplexType::isEmpty() const
             && d->mElements.isEmpty() && d->mBaseTypeName.isEmpty() && d->mArrayType.isEmpty();
 }
 
+bool ComplexType::operator==(const ComplexType &other) const
+{
+    return (XSDType::operator==(other)
+            && nameSpace() == other.nameSpace()
+            // ComplexType:
+            && baseDerivation() == other.baseDerivation() && baseTypeName() == other.baseTypeName()
+            && arrayType() == other.arrayType() && elements() == other.elements()
+            && attributes() == other.attributes() && attributeGroups() == other.attributeGroups());
+    // Note: Ignoring XmlElement::annotations(),
```

Comment:
This part of the note should move to XmlElement, it's `XmlElement::operator==` which (rightfully) ignores `annotations()`